src/Machine.Specifications.Runner.VisualStudio/Machine.Specifications.Runner.VisualStudio.csproj

```diff
@@ -16,17 +16,17 @@
     <IncludeBuildOutput>false</IncludeBuildOutput>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetFramework)'=='netcoreapp2.1'">
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
     <DefineConstants>NETSTANDARD</DefineConstants>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Mono.Cecil" Version="[0.10-*, 0.11)" />
-    <PackageReference Include="Machine.Specifications" Version="[0.11.0,2.0.0)" PrivateAssets="All" />
+    <PackageReference Include="Machine.Specifications" Version="1.*" PrivateAssets="All" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.1'">
-    <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="[15.0.0,17.0.0)" PrivateAssets="All" />
+  <ItemGroup Condition="'$(TargetFramework)'!='net452'">
+    <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="16.*" PrivateAssets="All" />
```

Comment:
Let's remove the conditions and just have a single package reference for all frameworks. I was wrong earlier, we only needed to support v11 for `net35`, which technically MSpec still supports. However the min version for the VS runner is `net452` and v16 works. 

Can you also set the versions of the ObjectModel dll to be `16.6.1`? That's the version xunit is using so I feel comfortable using that version.

Reply:
Why not use 16.10.0? Does xunit need to be updated!

Reply:
FYI https://github.com/xunit/visualstudio.xunit/pull/273

Reply:
![image](https://user-images.githubusercontent.com/1986660/119802653-9fe81d00-bed6-11eb-87b4-4b7d8b8d3873.png)
1. What do we build with? Product version 1.6.1 (but 16.* would be better)
2. What do we test with? Product version 1.6.10
3. What do we demand in the nuget? nothing (except Ceil)
4. What do we reference in the shipped dll? File version 15

Microsoft.VisualStudio.TestPlatform.ObjectModel, Version=15.0.0 dll, it might mean 
Microsoft.TestPlatform.ObjectModel, Version=16.x. nuget
![image](https://user-images.githubusercontent.com/1986660/119802949-e9d10300-bed6-11eb-923d-00cb635eaf4a.png)

Runner may or maynot include the Microsoft.VisualStudio.TestPlatform.ObjectModel.dll. If you load all assemblies, in a test, "v15" might not be found

TL;DR; we should remove v11.

Not sure we should "copy" xunit

Reply:
https://github.com/nunit/nunit3-vs-adapter/blob/master/src/NUnitTestAdapter/NUnit.TestAdapter.csproj#L53